_site/docs/architecture/queues.md

```diff
@@ -56,56 +56,44 @@ Otherwise, the execution request's properties must have exactly the same set of
 
 In this example the scheduler declares a GPU queue and CPU queue:
 ```
-redis_shard_backplane_config: {
-  provisioned_queues: {
-    queues: [
-      {
-        name: "gpu_queue"
-        platform: {
-          properties: [{name: "gpu" value: "1"}]
-        }
-      },
-      {
-          name: "cpu_queue"
-      }
-    ]
-  }
-}
+backplane:
+  queues:
+    - name: "cpu"
+      allowUnmatched: true
+      properties:
+        - name: "min-cores"
+          value: "*"
+        - name: "max-cores"
+          value: "*"
+    - name: "gpu"
+      allowUnmatched: true
+      properties:
+        - name: "gpu"
+          value: "1"
 ```
 
 ### Worker Example
 
 Queues are defined similarly on Workers:
 
 ```
-redis_shard_backplane_config: {
-  provisioned_queues: {
-    queues: [
-      {
-        name: "gpu_queue"
-        platform: {
-          properties: [{name: "gpu" value: "1"}]
-        }
-      },
-      {
-          name: "cpu_queue"
-      }
-    ]
-  }
-}
+backplane:
+  queues:
+    - name: "cpu"
+      allowUnmatched: true
+      properties:
+        - name: "min-cores"
+          value: "*"
+        - name: "max-cores"
+          value: "*"
+    - name: "gpu"
+      allowUnmatched: true
+      properties:
+        - name: "gpu"
+          value: "1"
 ```
 
-In addition to the queue declaration, the worker must specify from which queue tasks are obtained (otherwise the worker process will refuse to start):
```

Comment:
Is there going to be a replacement for this functionality when v2 is released?

Reply:
That was an incorrect documentation. The platform in dequeue match settings was actually not being used hence it was removed. The queue definitions are sufficient to match worker to a pool of workers.